link/perf_event.go

```diff
@@ -228,7 +245,6 @@ func openTracepointPerfEvent(tid uint64) (*internal.FD, error) {
 // and joined onto base. Returns error if base no longer prefixes the path after
 // joining all components.
 func uint64FromFile(base string, path ...string) (uint64, error) {
-
```

Comment:
Does your editor remove these automatically? There are a few of these removals strewn across your patches. I leave them in on purpose to provide some visual breathing room, please leave them in place in functions you don't functionally touch in the patch. :slightly_smiling_face:

Reply:
Sorry, it's my brain linter 💯 :D 

Thanks for the review, I'll fix everything later this afternoon